src/Fixer/FunctionNotation/NativeFunctionInvocationFixer.php

```diff
@@ -136,6 +152,14 @@ protected function applyFix(\SplFileInfo $file, Tokens $tokens)
                 continue;
             }
 
+            // in_array() will only be optimized if the third argument is true
```

Comment:
re-reading php-src, I think this can be removed, the optim occurs when the 3rd arg is missing, when it's `true` or `false` also.
Thanks for the PR BTW!

Reply:
You are welcome. Changed in 3cfc483e79e92f3f6fa9dcfd7ac763bdda9fc682.